storm-client/src/jvm/org/apache/storm/messaging/netty/StormServerHandler.java

```diff
@@ -51,14 +48,13 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
             server.received(msg, channel.remoteAddress().toString(), channel);
         } catch (InterruptedException e) {
             LOG.info("failed to enqueue a request message", e);
-            failureCount.incrementAndGet();
         }
     }
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
         try {
-            LOG.error("server errors in handling the request", cause);
+            LOG.error("server errors in handling the request to " + ctx.channel().remoteAddress().toString(), cause);
```

Comment:
change "to" to "from" since this is an `ChannelInboundHandlerAdapter`?

Reply:
Actually, I found (earlier) that passing parameter in the LOG.error does not handle the exception properly (i.e. stack trace is not printed). This maybe because we use an older version of SL4J. So the error above with {} will print only the message text but not stack. But if there is only one argument then the stack trace is printed. Here is a link that refers to sl4j version where this was fixed http://www.slf4j.org/faq.html#paramException

Reply:
Ignore previous comment. I just tested this in a storm test and parameters work in LOG.error correctly as expected.